EIPS/eip-875.md

```diff
@@ -1,9 +1,9 @@
 ---
 eip: 875
-title: A better NFT standard
+title: Simplier NFT standard with batching and native atomic swaps
 author: Weiwu Zhang <a@colourful.land>, James Sangalli <j.l.sangalli@gmail.com>
 discussions-to: https://github.com/ethereum/EIPs/issues/875
-status: Draft
+status: Final
```

Comment:
I don't think this can change to Final before going through a Last Call.

Reply:
Correct.

Reply:
@James-Sangalli I suggest you change this back to Draft in order to merge this change.